.travis.yml

```diff
@@ -12,8 +12,6 @@ addons:
 script:
   - bash keep_alive.sh &
   - export NODE_ENV=production
-  - mkdir ~/.kicad_plugins
-  - pushd ~/.kicad_plugins ; git clone https://github.com/kitspace/InteractiveHtmlBom.git ; cd InteractiveHtmlBom ; git checkout kitspace-config ; popd
```

Comment:
I think the `;` will swallow errors, maybe use `&&` here?

Reply:
I've changed the way this is done now -- the IBOM stuff is pulled in a as a Git submodule instead of this way of getting at it.